src/components/TrackTile/PillarsTile/Pillar.tsx

```diff
@@ -268,7 +270,12 @@ export const Pillar: FC<PillarProps> = (props) => {
                   ]}
                 />
                 {hasMetGoal && (
-                  <View style={styles.pillarStarBackground}>
+                  <View
+                    style={[
+                      styles.pillarStarBackground,
+                      { backgroundColor: backdrop },
```

Comment:
Should this go before `styles.pillarStartBackground` to allow overriding if desired?